composer.json

```diff
@@ -18,8 +18,8 @@
     "behat/behat": "~3.1.0-rc2",
     "behat/mink-extension": "~2.0",
     "drupal/drupal-driver": "dev-master",
-    "symfony/dependency-injection": "~2.7",
-    "symfony/event-dispatcher": "~2.7"
+    "symfony/dependency-injection": "2.8.*",
+    "symfony/event-dispatcher": "2.8.*"
```

Comment:
@jhedstrom good point. So in my own project's composer file I can just override the symfony requirement since the drupal-extension is using ~? I thought I had tried this but composer kept complaining.